Products/feedfeeder/utilities.py

```diff
@@ -447,4 +447,8 @@ def updateWithRemoteFile(obj, link):
 
 def feedfolder_created_handler(feedfolder, event):
     """Update feed folder after its creation"""
-    feedfolder.restrictedTraverse("update_feed_items")()
+    try:
+        feedfolder.restrictedTraverse("update_feed_items")()
+    except Exception as ex:
+        print("Error while retrieving items for %s:" % feedfolder.getId())
```

Comment:
The existing code makes use of the logger. While not use it here?

```suggestion
        logger.exception("Error while retrieving items for %s:" % feedfolder.getId())
```